benchmarks/apps/ramses/ramses.py

```diff
@@ -0,0 +1,92 @@
+import os.path as path
+import reframe as rfm
+import reframe.utility.sanity as sn
+from benchmarks.modules.utils import SpackTest
+
+
+class Ramses_download_inputs(rfm.RunOnlyRegressionTest):
+
+    descr = 'Download Ramses input files'
+    executable = 'wget'
+    executable_opts = [
+            f'https://zenodo.org/record/7842140/files/data.tgz'  # noqa: E501
+            ]
+    local = True
+
+    @sanity_function
+    def validate_download(self):
+        return sn.assert_true(path.exists('data.tgz'))
+
+
+class RamsesMPI(SpackTest):
+
+    valid_systems = ['*']
+    valid_prog_environs = ['*']
```

Comment:
```suggestion
    valid_prog_environs = ['deault']
```
Related to comment on sphng